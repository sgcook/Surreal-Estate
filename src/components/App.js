import "../styles/app.css";
import React from "react";
import { Switch, Route } from "react-router-dom";
import Navbar from "./NavBar";
import Properties from "./Properties";
import AddProperty from "./AddProperty";

const App = () => {
  return (
    <div className="App">
      <Navbar />

      <Switch>
        <Route exact path="/" component={Properties} />

        <Route exact path="/add-property" component={AddProperty} />
      </Switch>
    </div>
  );
};

export default App;
