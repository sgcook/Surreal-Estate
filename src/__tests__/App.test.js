import { render, screen } from "@testing-library/react";
import App from "../components/App";

test("renders learn react link", () => {
  render(<App />);
  expect(screen.getByRole("heading", { level: 2 })).toHaveTextContent(
    "Surreal Estate"
  );
});
