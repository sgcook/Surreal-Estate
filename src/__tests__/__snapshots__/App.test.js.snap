// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <img
        alt="surrealEstateLogo"
        class="surrealEstateLogo"
        src="surrealEstateLogo.png"
      />
      <ul
        class="navbar-links"
      >
        <a
          class="navbar-links-item"
          href="/"
        >
          View Properties
        </a>
        <a
          class="navbar-links-item"
          href="/add-property"
        >
          Add Property
        </a>
      </ul>
    </div>
    <div
      class="properties-page"
    >
      Properties Page
      <div
        class="property-card"
      >
        I am a property card
        <h3>
          beautiful home
        </h3>
        <ul
          class="property-card__list"
        >
          <li
            class="type-ctiy"
          >
            bungalow - Leeds
          </li>
          <li
            class="bathrooms"
          >
            3
          </li>
          <li
            class="bedrooms"
          >
            2
          </li>
          <li
            class="price"
          >
            Â£20000
          </li>
        </ul>
        <p
          class="email-p"
        >
          <a
            class="email-link"
            href="mailto:joe@gmail.com"
          >
            Email
          </a>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
